README.md

```diff
@@ -193,3 +196,30 @@ Use this with any client interface such as [MoveIt!](https://moveit.ros.org/) or
 rosrun rqt_joint_trajectory_controller rqt_joint_trajectory_controller
 ```
 
+## Troubleshooting
+
+This section will cover some previously raised issues.
+
+1. **I started everything, but I cannot control the robot.**<br/>
+ The `External Control` program node from the URCap is not running on the robot. Make sure to
+ create a program containing this node on the robot and start it. Inside the ROS terminal you
+ should see the output `Robot ready to receive control commands.`<br/>
+ Note: When interacting with the teach pendant, or sending other primary programs to the robot, the
+ program will be stopped. On the ROS terminal you will see an output `Connection to robot
+ dropped, waiting for new connection`. In those cases, restart program execution (e.g. by pressing
+ the play button on the TP).<br>
+ In general, make your you did follow this approach:
+    1. Install and setup the `External Control` URCap as explained [above](#setting-up-a-ur-robot-for-ur_robot_driver) (also setup the IP address **of the ROS pc** inside the URCap's installation.)
+    2. Start the driver handing the IP **of the robot** as explained in the [quickstart](#quick-start)
+    3. Load and start the previously generated program on the TP.
+2. **When starting the program on the TP, I get an error "The connection to the remote PC could not
+   be established"** <br/>
+ Make sure, the IP address setup is correct, as described in the setup guides ([CB3 robots](ur_robot_driver/doc/install_urcap_cb3.md),
+ [e-Series robots](ur_robot_driver/doc/install_urcap_e_series.md))<br/>
+ Note: This error can also show up, when the ROS driver is not running.
+3. **When starting the program on the TP, I get a `C207A0` error.**<br/>
+ Most probably, the EtherNet/IP fieldbus is enabled in the robot's installation. Since the robot's
+ ethernet port is used for communicating with ROS (Node: EtherNet/IP != ethernet), you'll have to
+ deactivate this. See
```

Comment:
Is this actually true?

EtherNet/IP should tolerate additional devices on the same segment, so a switch could be used.

I understood the comments in #28 as if safety over EtherNet/IP was configured, so without there being an actual EtherNet/IP connection, the controller went into a fault mode, to prevent operation of the robot without the safety-system working properly.

Reply:
True, that doesn't hit the point, exactly.